Formula/cfitsio.rb

```diff
@@ -39,4 +40,18 @@ def install
       end
     end
   end
+
+  test do
+    cp Dir["#{pkgshare}/testprog/testprog*"], testpath
+    flags = (ENV.cflags || "").split + (ENV.cppflags || "").split + (ENV.ldflags || "").split
```

Comment:
@MikeMcQuaid There are 66 formulae with exactly this line (try `grep -rI 'ENV.cflags || ""' "$(brew --repository homebrew/core)"`); that's why I've been using it as a boilerplate. Any reason to change it to something else?